.github/workflows/stable.yml

```diff
@@ -29,13 +30,10 @@ jobs:
       - name: Get the newest rubygems version to rid ourselves of warnings
         run: gem update --system --no-document
 
-      - name: Install bundler
-        run: gem i bundler --no-document
-
-      - name: Install dependencies
-        run: |
-          bundle config set --local without benchmark
-          bundle install --jobs=3
-
       - name: Run tests
         run: bundle exec rake
+        if: matrix.ruby != 'truffleruby'
+
+      - name: Run specs (truffleruby)
+        run: bundle exec rake spec
```

Comment:
I'm not running `cucumber` on TruffleRuby because that doesn't work due to old cucumber using old aruba using childprocess which is not really maintained currently (https://github.com/enkessler/childprocess/issues/172).